src/astarte-client/models/Device/index.ts

```diff
@@ -16,7 +16,49 @@
    limitations under the License.
 */
 
-import { AstarteDeviceDTO } from '../../types';
+import validation from './validation';
```

Comment:
now the validation is pretty compact. Why not keep it in the same file? In my opinion it's usefull to see the data structure and how it's validated

Reply:
I moved the validation into the same file